packages/opentelemetry-propagation-jaeger/README.md

```diff
@@ -0,0 +1,37 @@
+# OpenTelemetry Propagation Jaeger
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+OpenTelemetry propagation Jaeger provide HTTP header propagation for systems that using Jaeger HTTP header format.
+Example of usage:
+```javascript
+const { NodeTracer } = require('@opentelemetry/node');
```

Comment:
This is using the old style `NodeTracer` which has been replaced with `NodeTracerRegistry`